client/src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,31 @@
+.modal {
+  height: 100vh;
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.4);
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: 0.5s;
+  opacity: 0;
+  pointer-events: none;
+}
+
+.modal.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.content {
+  padding: 20px;
+  border-radius: 12px;
+  background-color: #fff;
+  width: 50vh;
+  transform: scale(0.5);
+  transition: 0.4s all;
+}
+.content.active {
```

Comment:
If youuse SCSS, use its power for nested classes, e.g. `.content { .active { ... } }`